client/app/components/dynamic-table/dynamic-table.css

```diff
@@ -1,3 +0,0 @@
-th.sortable-column {
```

Comment:
We're using this in other places too -- like `alerts-list.html`.

Reply:
1. It's not removed, just renamed to `.less`.
2. I was thinking that this file contains component-specific CSS; it there are some styles shared with other components or some common CSS - it should be moved to the component or `assets/less` respectively. Will do it a bit later.

Reply:
1. But in the `less` file it's scoped to the dynamic-table container, isn't it?
2. True, my bad probably.

Reply:
Moved that styles to `assets/less/inc/table.less` - hope it's better place for it.